packages/hub/services/web3-storage.ts

```diff
@@ -0,0 +1,24 @@
+import { Web3Storage as Storage } from 'web3.storage';
+import { getFilesFromPath } from 'web3.storage';
+import { Filelike } from 'web3.storage/dist/src/lib/interface';
+import config from 'config';
+
+export default class Web3Storage {
+  async upload(path: string, name: string) {
+    const client = new Storage({
+      token: config.get('web3storage.token'),
+    });
+
+    let files = await getFilesFromPath(path);
```

Comment:
@jurgenwerk if this is meant to work on the hub, can `getFilesFromPath` actually get the files? It feels like it can't, and we might need to use `web3Storage.put`?

Reply:
Does koa/formidable do some work in the background that makes uploaded files have a path?

Reply:
It looks like the files are uploaded to /tmp path. I didn't manage to find a way where we could stream the file upload directly to web3 storage, since the Web3.Storage client's put method only accepts [an array of File objects](https://docs.web3.storage/how-tos/store/#preparing-files-for-upload)